modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -0,0 +1,98 @@
+
+MS2-Randomize:: When this header is sent as true, the mocking service auto-generates a payload response if no example is defined for the response.
++
+For example, suppose that an API specification contained the definitions of this type:
++
+----
+types:
+  Person:
+    properties:
+      id: integer
+      name: string
+----
++
+If you did not provide an example, but you used the header MS2-Randomize, set to `true`, with the mocking service, the payload of the returned message would contain an example with random values that matched the data types, like this:
++
+----
+{
+  "id": 98358,
+  "name": "OVCKr"
+}
+----
++
+Moreover, the header respects more detailed definition of types. For example, you might change your type definition to look like this:
++
+----
+types:
+  Person:
+    properties:
+      id:
+        type: integer
+        minimum: 3
+        maximum: 30
+        multipleOf: 3
+      name: string
+----
++
+If MS2-Randomize were set to `true`, the payload in the response from the mocking service might look like this, with the value of `id` conforming to the constraints:
++
+----
+{
+  "id": 9,
+  "name": "rul68"
+}
+----
+
+MS2-Status-Code:: Specifies the non-error code and the error code that the mocking service can choose from for the return message. Separate the two status codes with a comma. For example, you might want the mocking service to return a 201 message, if there is no error, or a 401 status code, if there is an error; in this case, you would specify `201,401` for the value.
++
+If you use this header and you do not provide a non-error status code, the mocking service throws an error if the API does not return an error. Also, if you use this header and you do not provide an error status code, the mocking service throws an error if the API returns an error.
++
+If you do not use this header, the mocking service behaves as follows:
++
+* If the API returns a non-error status code, the mocking service returns the first non-error status code that is listed for the method.
+* If the API returns an error status code, the mocking service returns the first error status code that is listed for the method.
+
+MS2-Strict-Model-Validations:: This header can be helpful when you are simulating calls to an API with a specification that was written before API Designer switched to its stricter parser on January 10, 2019.
++
+When you set this parameter to `false`, you can use the mocking service even when the API specification contains errors.
++
+When you set this parameter to `true`, the mocking service responds only by returning the first error status code that is defined in the specification.
```

Comment:
In case of setting this header to `true`, the mocking service will validate the spec before answering. So, if there is an error on the spec, Mocking Service will fail with that error, otherwise, Mocking Service will work as usual

Reply:
Another minor comment.. you use the word parameter, perhaps using header instead would be better here

Reply:
Fixed.